### PR TITLE
style: Database connect UI Polish - Step 1

### PR DESCRIPTION
**Pull request by lyndsiWilliams**
### SUMMARY
Added styling to match the [figma specifications](https://www.figma.com/file/NbuTOMaUdv2GCBoDyihdLw/Database-Connection?node-id=1698%3A394658) for step 1 of the database connection form.

### SCREENSHOTS
step1-dbconnect-fix

### TESTING INSTRUCTIONS
From the Superset home page:
- Hover over "Data" in the upper navigation bar
- Click "Database"
- Click the "+ Database" button below the navigation bar and to the right
- Observe the styling of "Step 1" of the database connection form

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx [hunk 1]
```diff
@@ -624,6 +625,12 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
                 />
                 {renderPreferredSelector()}
                 {renderAvailableSelector()}
+                <Alert
```
Comment (hughhhh) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx:628]: Remove this @AAfghahi already has this set in on the feature branch

Comment (lyndsiWilliams) [reply superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx]: Oh awesome! Removed in [`this commit`](https://github.com/apache/superset/pull/15119/commits/4cb8935b0b03aec55b4c5d699eafcd4022906340).

**Comment by betodealmeida** [comment]
The text under the DB icons is slightly skewed towards the left, is that something you're planning to fix in this PR?

**Comment by lyndsiWilliams** [comment]
> The text under the DB icons is slightly skewed towards the left, is that something you're planning to fix in this PR?

Oops! Yes, missed that bit when I brought the LinesEllipsis in. Thanks for that catch! Fixed in [`this commit`](https://github.com/apache/superset/pull/15119/commits/59e28e1a939118bfcceecebdaacb58fd3ec0ee01).

**Review by betodealmeida — APPROVED** [review]
This looks great!

Can you post a screenshot showing the ellipsis?

**Comment by hughhhh** [comment]
Merging this is now so i can make a build with superset-shell

**Comment by lyndsiWilliams** [comment]
Thank you! Just updated the description with the ellipsis 😁

**Pull request merged**